pkg/util/mock-exec-run.go

```diff
@@ -0,0 +1,44 @@
+package util
+
+import (
+	"errors"
+	"time"
+)
+
+// MockRunner implements the methods of Runner interface
+type MockRunner struct {
```

Comment:
Since this will be inside mock package, we can rename this to `runner`

Can we have below?

```go
// runner is used to mock os/exec. 
// Expected data is set against its
// instance.
type runner struct {
  // output is the expected return 
  // value
  output            []byte

  // isError flags this instance
  // to return error
  isError bool
}

type builder struct {
  runner  *runner
}

func Builder() *builder {
  return &builder{runner: &runner{}}
}

func (b *builder) WithOutput(output string) *builder {
  b.runner.output = output.(byte)
  return b
}

func (b *builder) Error() *builder {
  b.runner.isError = true
  return b
}

func (b *builder) Build() *runner {
  return b.runner
}
```

Caller code will look like:
```go
import (
  mockexec "github.com/openebs/maya/pkg/exec/mock/v1alpha1"
)

func TestABC(t *testing.T) {
  // ...
  e: = mockexec.Builder().
            WithOutput(test.zpoolOutput).
            Build()
  abc, err := e.RunCombinedOutput(...)
}
```

Reply:
done